.github/workflows/nightly-test-integrations-1.15.x.yml

```diff
@@ -290,13 +290,16 @@ jobs:
           DD_ENV: ci
         run: datadog-ci junit upload --service "$GITHUB_REPOSITORY" $TEST_RESULTS_DIR/results.xml
 
+
   test-integrations-success:
     needs: 
     - setup
     - dev-build
     - generate-envoy-job-matrices
     - envoy-integration-test
     - upgrade-integration-test
+    # TODO: AC6 broken for 1.15
+    # - peering_commontopo-integration-test
```

Comment:
Will you create a follow up ticket?

Reply:
I'm gonna give it an hour or two as part of this PR, and if not, will punt to a new ticket

Reply:
I gave it a try, but couldn't come up with a good way to fix it.

Seems like the problem is with:

```
				ct.Assert.UpstreamEndpointStatus(t, client, fmt.Sprintf("failover-target~%d~%s", i, clusterPrefix(u, clu.Datacenter)), "HEALTHY", 1)
```

In 1.15, that string is `failover-target~%s`, i.e., missing the ordinal for some reason. We'd want to dig into why that is.

But furthermore, I can't see a simple way to do something like `if version >= 1.16`. `deployer` doesn't really know which version of Consul it's running; it just runs whatever container is in `hashicorp/consul:local` AFAICT. And I couldn't figure out a way to query the server for that info. 

Another approach would be to check against a set of envoy cluster names (or a regex). This assertion goes to `libassert` eventually, and that function uses jq (🙄 ) to check the returned JSON. 

Seemed not worth the effort, for now.